### ISSUE TITLE
Waaruit wordt de gebruiker bepaald in Open Zaak? (t.a.v. de audit trail)

### ISSUE THREAD

**Issue by OlavAllema** [open-zaak/open-zaak#1334]
**General information**
Open Zaak version: 1.6.0

**Describe the bug**
In de audit trail zien we bij het verwijderen van een document dat Open Zaak 'system' aangeeft als gebruiker, terwijl de actie wordt gedaan door een ingelogde gebruiker.

**To Reproduce**
1. Kies een zaak met een document
2. Verwijder het document
3. Bekijk de audit trail

**Expected behavior**
We verwachten dat in de audit trail de gebruiker wordt aangegeven als degene die het document heeft verwijderd.
![image](https://user-images.githubusercontent.com/75848616/217856131-7d3582d3-7355-4c5c-941d-3f6c5ee792b7.png)

**Comment by joeribekker** [comment]
In de API geef je dat mee als attribuut in het JWT

**Comment by sergei-maertens** [comment]
Klopt, alles wordt gerelateerd aan de `user_id/user_representation` en `client_id` in de JWT claims.
